common/src/main/java/org/apache/celeborn/common/network/TransportContext.java

```diff
@@ -93,35 +97,60 @@ public TransportContext(TransportConf conf, BaseMessageHandler msgHandler) {
     this(conf, msgHandler, false, false, null);
   }
 
+  public TransportClientFactory createClientFactory(List<TransportClientBootstrap> bootstraps) {
+    return new TransportClientFactory(this, bootstraps);
+  }
+
   public TransportClientFactory createClientFactory() {
-    return new TransportClientFactory(this);
+    return createClientFactory(Collections.emptyList());
   }
 
   /** Create a server which will attempt to bind to a specific host and port. */
   public TransportServer createServer(String host, int port) {
-    return new TransportServer(this, host, port, source);
+    return new TransportServer(this, host, port, source, msgHandler, Collections.emptyList());
+  }
+
+  public TransportServer createServer(
```

Comment:
Could we add these create methods when actually use them?

Reply:
Removed the methods which were not being used now.